tools/shoestring/shoestring/healthagents/rest_https_certificate.py

```diff
@@ -27,39 +27,63 @@ def _openssl_run_sclient_verify(hostname, test_args):
 	] + test_args, command_input='Q', show_output=False)
 
 	# verify the certificate is valid
+	certificates = []
+	collect_certificates = False
```

Comment:
"cert" and "certificate" are being used interchangeably in variable names, consider picking one and using it throughout